cwrap/prototype.py

```diff
@@ -223,7 +223,8 @@ def __call__(self, *args):
             # ArgumentError.message will look like this
             #   `argument 4: <type 'exceptions.TypeError'>: wrong type`
             # The only useful information here is the index of the argument 
-            tokens = re.split("[ :]", err.message)
+            errMsg = err.message if hasattr(err, "message") else str(err)
+            tokens = re.split("[ :]", errMsg)
             argidx = int(tokens[1]) - 1  # it starts from 1
             raise TypeError((
```

Comment:
Could you also change this to `raise TypeError(...) from err` ?

Reply:
oh sure - wasn't familiar with that, looks cool ^^

Reply:
it is done!